### DISCUSSION TITLE
Export labels as img

### DISCUSSION THREAD

**Discussion by volodos621** [wkentaro/labelme#1057] [Q&A / Help] [answered]
Hi,

Is-there a way to export labels as images (not the mask)?

I have a set of images and json, with some classes, and would like at least to export each label independently. For instance:
class_X_1.png
class_X_2.png
etc

Thank you for your help

**Comment by jkfindeisen** [comment]
See also #812

**Comment by wkentaro** [comment, marked as answer]
See https://github.com/wkentaro/labelme/discussions/1100 for solution.
